erroranalysis/erroranalysis/analyzer/error_analyzer.py

```diff
@@ -40,6 +40,10 @@ class BaseAnalyzer(ABC):
     :type feature_names: numpy.array or list[]
     :param categorical_features: The categorical feature names.
     :type categorical_features: list[str]
+    :param metadata_columns: The set of columns that are not passed
+        to the model or explainers. These columns can be used for
+        other analyses.
```

Comment:
Can you elaborate more on what other analysis is intended?

Reply:
I don't necessarily want to keep a running list of what we do with this in 10 different places. It's metadata, i.e., it provides data on your data and you can use it to analyze your data with more information than what you just get from your data. One such application is disaggregated analysis, but you can also use it for error analysis, or in fact for causal analysis. In the future we may have others, too.

Reply:
I think a 'such as' phrase could be useful here, but agree that it doesn't need to be length.

Reply:
Added!